### PR TITLE
CORE-4950: Move local hosted identities out of tests

### PR DESCRIPTION
**Pull request by yift-r3**
This PR:
1. Add RPC command to read a certificate chain.
2. Change the save certificate to accept a certificate chain.
3. Move the holding identity from the `p2p.test` to the `p2p` package.

The runtime PR is in https://github.com/corda/corda-runtime-os/pull/1427

### CODE REVIEW

File: data/avro-schema/src/main/resources/avro/net/corda/data/certificates/rpc/request/ImportCertificateRpcRequest.avsc [hunk 1]
```diff
@@ -11,7 +11,7 @@
     },
     {
       "name": "certificate",
```
Comment (charlieR3) [inline data/avro-schema/src/main/resources/avro/net/corda/data/certificates/rpc/request/ImportCertificateRpcRequest.avsc]: What is the intention here? A string of concatenated certs? Or should it be an array of Strings? If the comment is changing the plural maybe the name should also? i.e. `certificate` -> `certificates`

File: data/avro-schema/src/main/resources/avro/net/corda/data/certificates/rpc/response/CertificateRetrievalRpcResponse.avsc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "type": "record",
+  "name": "CertificateRetrievalRpcResponse",
+  "namespace": "net.corda.data.certificates.rpc.response",
+  "doc": "RPC operations response with a certificate chain.",
+  "fields": [
+    {
+      "name": "certificate",
+      "doc": "The certificates in PEM format.",
+      "type": [
+        "string",
```
Comment (charlieR3) [inline data/avro-schema/src/main/resources/avro/net/corda/data/certificates/rpc/response/CertificateRetrievalRpcResponse.avsc:11]: Same question as above. Is this intended to be a concatenated string of certs? I think we're using arrays in others places.

File: data/avro-schema/src/main/resources/avro/net/corda/data/certificates/rpc/request/ImportCertificateRpcRequest.avsc (hunk 1 above)
Comment (yift-r3) [reply data/avro-schema/src/main/resources/avro/net/corda/data/certificates/rpc/request/ImportCertificateRpcRequest.avsc]: It should be a string of concatenated certificates.

**Review by charlieR3 — DISMISSED** [review]

**Review by charlieR3 — APPROVED** [review]
LGTM

**Pull request merged**